cmd/test

```diff
@@ -57,11 +57,15 @@ basic() {
   done
 
   echo "...checking if LDOP is currently running..."
-  PS=$(${CONF_DIR}/adop compose ps -q | wc -l)
+  PS=$(${CONF_DIR}/adop compose ps 2>/dev/null)
+
+  # Get all the output after the last occurance of *-\s
+  AFTER_HEADER=${PS##*----}
 
   if [ $F -eq 0 ]
   then
-    if [ $PS -gt 4 ]
+    # If there are any items in /adop compose ps...
+    if [ "${#AFTER_HEADER}" -gt "0" ] 
```

Comment:
This doesn't handle the case where the platform is paused, as stopping is not a permitted operation on a paused container.

Sample error output: ERROR: for elasticsearch  Cannot stop container 6c6cf14118ae2702ddf066f66805e505e79940a785a7df7ee2cc150549c8cd6a: Container 6c6cf14118ae2702ddf066f66805e505e79940a785a7df7ee2cc150549c8cd6a is paused. Unpause the container before stopping